From 83f9dbb9420b01be546b0255ac19d44785d93b37 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 21 Jun 2012 23:22:53 +0200
Subject: [PATCH] preset: don't look for preset files in /lib unless /usr is
 split off (cherry picked from commit
 b4bdfefac3fcf633aa0700a981d854cc49a9725b)

---
 src/shared/install.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/shared/install.c b/src/shared/install.c
index 2104c30..f443578 100644
--- a/src/shared/install.c
+++ b/src/shared/install.c
@@ -1685,7 +1685,9 @@ int unit_file_query_preset(UnitFileScope scope, const char *name) {
                                     "/etc/systemd/system.preset",
                                     "/usr/local/lib/systemd/system.preset",
                                     "/usr/lib/systemd/system.preset",
+#ifdef HAVE_SPLIT_USR
                                     "/lib/systemd/system.preset",
+#endif
                                     NULL);
         else if (scope == UNIT_FILE_GLOBAL)
                 r = conf_files_list(&files, ".preset",
